.doc_gen/templates/zonbook/sdk_chapter_template.xml

```diff
@@ -8,103 +8,151 @@
+                            <title id="{{.ExampleId}}_{{$.LanguageSlug}}_topic.titleabbrev">{{.TitleAbbrev}}</title>
+                            <titleabbrev id="{{.ExampleId}}_{{$.LanguageSlug}}_topic.titleabbrev">{{.TitleAbbrev}}</titleabbrev>
+                        </info>
+                        {{- if or .Synopsis .SynopsisList}}
+                            {{- $synopsis_prefix := "The following code example shows how to"}}
+                            {{- $synopsis_sep := " "}}
+                            {{- if not .Synopsis}}
+                                {{- $synopsis_sep = ":"}}
+                            {{- end}}
+                        <para>{{print $synopsis_prefix $synopsis_sep .Synopsis}}</para>
+                            {{- if .SynopsisList}}
+                        <itemizedlist>
+                            {{- range $synitem := .SynopsisList}}
+                            <listitem><para>{{$synitem}}</para></listitem>
+                            {{- end}}
+                        </itemizedlist>
+                            {{- end}}
                         {{- end}}
-                    </itemizedlist>
+                        {{- if .GuideTopic.Url}}
+                        <para>For more information, see <ulink {{.GuideTopic.DocType}} url="{{.GuideTopic.Url}}">{{.GuideTopic.Text}}</ulink>.</para>
                         {{- end}}
+                        <xi:include href="file://AWSShared/code-samples/docs/{{.ExampleId}}_{{$.Language}}.xml"></xi:include>
+                    </section>
                     {{- end}}
-                    {{- if .GuideTopic.Url}}
-                    <para>For more information, see <ulink {{.GuideTopic.DocType}} url="{{.GuideTopic.Url}}">{{.GuideTopic.Text}}</ulink>.</para>
-                    {{- end}}
-                    <xi:include href="file://AWSShared/code-samples/docs/{{.ExampleId}}_{{$.Language}}.xml"></xi:include>
-                </section>
-                {{- end}}
-            </collapsible>
+                </collapsible>
+            </section>
             {{- end}}
             {{- end}}
         </section>
         {{- end}}
         {{- if .ApiExamples}}
     </section>
     {{- end}}
-</chapter>
\ No newline at end of file
+    {{- if .CrossServiceExamples}}
+    <section id="{{.LanguageSlug}}_code_examples_cross_service" role="topic">
+        <info>
+            <title id="{{.LanguageSlug}}_code_examples_cross_service.title">Cross-service examples using {{.SdkEntity.Short}}</title>
+            <titleabbrev id="{{.LanguageSlug}}_code_examples_cross_service.titleabbrev">Cross-service</titleabbrev>
+            <abstract>
+                <para>The following sample applications use {{.SdkEntity.Long}}
+                    to combine multiple &AWS; services.</para>
+            </abstract>
+        </info>
+        <para>The following sample applications use {{.SdkEntity.Long}} to combine
+            multiple &AWS; services.</para>
```

Comment:
<para>The following sample applications use the {{.SdkEntity.Long}}
                to work across multiple &AWS; services.</para>

Reply:
Done